Missing scule dependency in demo-store template

While using the demo-store template, I noticed that `scule` is imported directly in several files but is not declared as a dependency of the corresponding package.

For example, in `pages/[...all].vue`:

```ts
import { pascalCase } from "scule";
```

However, `scule` is not listed in the package's `dependencies` or `peerDependencies`.

As a result, TypeScript reports:

```
Cannot find module 'scule' or its corresponding type declarations.
```

The project may still work if `scule` is installed transitively by another dependency, but relying on transitive dependencies is fragile and may break depending on the package manager (e.g. pnpm) or module resolution settings.

**Expected behavior**

`scule` should be declared as a direct dependency of the package that imports it, or the import should be replaced if it is no longer required.

--- a/templates/vue-demo-store/package.json
+++ b/templates/vue-demo-store/package.json
@@ -29,6 +29,7 @@
     "@vueuse/nuxt": "14.3.0",
     "defu": "6.1.7",
     "requrl": "3.0.2",
+    "scule": "1.3.0",
     "sitemap": "8.0.0",
     "unocss": "66.7.4",
     "vue": "3.5.39"
@@ -39,10 +40,10 @@
     "@nuxtjs/i18n": "10.4.0",
     "@shopware/api-gen": "canary",
     "nuxt": "4.4.8",
-    "typescript": "5.9.3",
-    "vite": "8.1.3",
     "oxfmt": "0.55.0",
-    "oxlint": "1.70.0"
+    "oxlint": "1.70.0",
+    "typescript": "5.9.3",
+    "vite": "8.1.3"
   },
   "engines": {
     "node": "^20.x || ^22.x || ^24.x"

--- a/templates/vue-starter-template/package.json
+++ b/templates/vue-starter-template/package.json
@@ -37,22 +37,23 @@
     "@shopware/helpers": "canary",
     "@shopware/nuxt-module": "canary",
     "@shopware/unocss-design-tokens-layer": "canary",
-    "@unocss/nuxt": "66.7.4",
     "@unocss/core": "66.7.4",
+    "@unocss/nuxt": "66.7.4",
     "@vueuse/core": "14.3.0",
     "@vueuse/nuxt": "14.3.0",
     "nuxt": "4.4.8",
+    "scule": "1.3.0",
     "unocss": "66.7.4",
     "vue": "3.5.39"
   },
   "devDependencies": {
     "@shopware/api-gen": "canary",
     "@vue/compiler-sfc": "3.5.39",
+    "oxfmt": "0.55.0",
+    "oxlint": "1.70.0",
     "typescript": "5.9.3",
     "vite": "8.1.3",
-    "vue-tsc": "3.3.6",
-    "oxfmt": "0.55.0",
-    "oxlint": "1.70.0"
+    "vue-tsc": "3.3.6"
   },
   "overrides": {
     "glob": "^13",

